bundle/terranova/tnresources/alert.go

```diff
@@ -0,0 +1,59 @@
+package tnresources
+
+import (
+	"context"
+
+	"github.com/databricks/cli/bundle/config/resources"
+	"github.com/databricks/databricks-sdk-go"
+	"github.com/databricks/databricks-sdk-go/service/sql"
+)
+
+type ResourceAlert struct {
+	client *databricks.WorkspaceClient
+}
+
+// New initializes a ResourceAlert with the given client.
+func (*ResourceAlert) New(client *databricks.WorkspaceClient) *ResourceAlert {
+	return &ResourceAlert{client: client}
+}
+
+// PrepareState converts bundle config to the SDK type.
+func (*ResourceAlert) PrepareState(input *resources.Alert) *sql.AlertV2 {
+	return &input.AlertV2
+}
+
+// DoRefresh reads the alert by id.
+func (r *ResourceAlert) DoRefresh(ctx context.Context, id string) (*sql.AlertV2, error) {
+	return r.client.AlertsV2.GetAlertById(ctx, id)
+}
+
+// DoCreate creates the alert and returns its id.
+func (r *ResourceAlert) DoCreate(ctx context.Context, config *sql.AlertV2) (string, *sql.AlertV2, error) {
+	request := sql.CreateAlertV2Request{
+		Alert: *config,
+	}
+	response, err := r.client.AlertsV2.CreateAlert(ctx, request)
+	if err != nil {
```

Comment:
Suggestion: 	`if err != nil || response == nil`

outer code will catch an empty id case and raise an error but crash is prevented.

Reply:
Thanks. As long as there are no side effects, returning an empty ID sounds reasonable. I've accepted the suggestion to keep this code consistent with the other bits.